EventFilter/CTPPSRawToDigi/interface/RawToDigiConverter.h

```diff
@@ -69,29 +69,27 @@ class RawToDigiConverter {
     TotemVFATStatus status;
   };
 
-  unsigned char verbosity;
+  const unsigned char verbosity;
```

Comment:
Fantastic! And long-awaited. Could we jump on this opportunity to sanitise a bit the class members and introduce trailing `_` to all these?

Reply:
> Fantastic! And long-awaited. Could we jump on this opportunity to sanitise a bit the class members and introduce trailing `_` to all these?

@forthommel feel free to take this commit and adapt it into another more complete PR of yours, if you think so.
Please take into account that there will be that Totem special run in less than one month from now, for which some of that code will need to be backported to 13_0_X (maybe a dedicated release): I would defer too much invasive updates for when those backports will get concluded